internal/client/cluster/iam_policy/iam_resource.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package iamclusterclient
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/client/transport"
+	clustermodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/cluster"
+	iammodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/iam_policy"
+	clusteriammodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/iam_policy/cluster"
+)
+
+// New creates a new cluster IAM policy resource service API client.
+func New(transport *transport.Client) ClientService {
+	return &Client{Client: transport}
+}
+
+/*
+Client for cluster IAM policy resource service API.
+*/
+type Client struct {
+	*transport.Client
+}
+
+// ClientService is the interface for Client methods.
+type ClientService interface {
+	ManageV1alpha1ClusterIAMPolicyGet(fn *clustermodel.VmwareTanzuManageV1alpha1ClusterFullName) (*clusteriammodel.VmwareTanzuManageV1alpha1ClusterGetClusterIAMPolicyResponse, error)
+
+	ManageV1alpha1ClusterIAMPolicyPatch(request *clusteriammodel.VmwareTanzuManageV1alpha1ClusterPatchClusterIAMPolicyRequest) (*clusteriammodel.VmwareTanzuManageV1alpha1ClusterPatchClusterIAMPolicyResponse, error)
+
+	ManageV1alpha1ClusterIAMPolicyUpdate(fn *clustermodel.VmwareTanzuManageV1alpha1ClusterFullName, request *iammodel.VmwareTanzuCoreV1alpha1PolicyIAMPolicy) (*clusteriammodel.VmwareTanzuManageV1alpha1ClusterUpdateClusterIAMPolicyResponse, error)
+}
+
+/*
+  ManageV1alpha1ClusterIAMPolicyGet gets policy for a cluster.
```

Comment:
gets all iam policies scoped to a cluster. 

On the similar lines can you do the changes  for the below function comments as well

Reply:
Done at all places